src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -151,6 +153,44 @@ const WorkspaceAccessError = () => {
   ])
 }
 
+const checkBucketAccess = withErrorReporting('Error checking bucket access', async (signal, namespace, name) => {
+  try {
+    await Ajax(signal).Workspaces.workspace(namespace, name).checkBucketReadAccess()
+    return true
+  } catch (error) {
+    if (error.status === 403) {
+      notify('error', div([
+        'The Google Bucket associated with this workspace is currently unavailable. This should be resolved shortly. If this persists for more than an hour, please ',
+        link({
+          onClick: () => contactUsActive.set(true),
```

Comment:
Is there an `onDismiss` that you can pass `link`? That way when a user clicks on the Contact Us, the toast is dismissed. Currently Contact us shows up under the toast:
<img width="509" alt="Screen Shot 2019-05-10 at 10 50 49 AM" src="https://user-images.githubusercontent.com/2395211/57536117-85ec3980-7311-11e9-83bf-f414c4bb6e57.png">